<!DOCTYPE html>
    <html>
    <head>
        <meta charset="UTF-8">
        <title>Pacific Informatics&lt;&sol;br&gt;Gene Panel DNAseq variant analysis</title>
        <style>
/* From extension vscode.github */
/*---------------------------------------------------------------------------------------------
 *  Copyright (c) Microsoft Corporation. All rights reserved.
 *  Licensed under the MIT License. See License.txt in the project root for license information.
 *--------------------------------------------------------------------------------------------*/

.vscode-dark img[src$=\#gh-light-mode-only],
.vscode-light img[src$=\#gh-dark-mode-only] {
	display: none;
}

</style>
        
        <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/Microsoft/vscode/extensions/markdown-language-features/media/markdown.css">
<link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/Microsoft/vscode/extensions/markdown-language-features/media/highlight.css">
<style>
            body {
                font-family: -apple-system, BlinkMacSystemFont, 'Segoe WPC', 'Segoe UI', system-ui, 'Ubuntu', 'Droid Sans', sans-serif;
                font-size: 14px;
                line-height: 1.6;
            }
        </style>
        <style>
.task-list-item {
    list-style-type: none;
}

.task-list-item-checkbox {
    margin-left: -20px;
    vertical-align: middle;
    pointer-events: none;
}
</style>
        
    </head>
    <body class="vscode-body vscode-light">
        <h1 id="pacific-informaticsgene-panel-dnaseq-variant-analysis">Pacific Informatics</br>Gene Panel DNAseq variant analysis</h1>
<p><strong><em>PHYSICIAN REPORT</em></strong></p>
<hr>
<h2 id="table-of-contents">Table of Contents</h2>
<ul>
<li><a href="#pacific-informaticsgene-panel-dnaseq-variant-analysis">Pacific InformaticsGene Panel DNAseq variant analysis</a>
<ul>
<li><a href="#table-of-contents">Table of Contents</a></li>
<li><a href="#sample-report">Sample report</a>
<ul>
<li><a href="#1-genes-assayed">1. Genes Assayed</a></li>
<li><a href="#2-relevant-variant-details">2. Relevant Variant Details</a></li>
</ul>
</li>
<li><a href="#citations">Citations</a></li>
</ul>
</li>
</ul>
<h2 id="sample-report">Sample report</h2>
<p><em><strong>Date: April 28th 2023</strong></em></p>
<h3 id="1-genes-assayed">1. Genes Assayed</h3>
<p><strong><em>Genes Assayed for the Detection of DNA Sequence Variants</em></strong></p>
<blockquote>
<p>MPL, NRAS, RET, PTEN, FGFR2, HRAS, ATM, KRAS, PTPN11, HNF1A, FLT3 ,BRCA2, RB1, AKT1, IDH2, TSC2, CDH1, TP53, ERBB2, BRCA1, SMAD4, STK11, GNA11, JAK3, ALK, IDH1, ERBB4, SRC, GNAS, SMARCB1, VHL, MLH1, CTNNB1, PIK3CA, FGFR3, PDGFRA, KIT, KDR, FBXW7, APC, CSF1R, NPM1, EGFR, MET, SMO, BRAF, EZH2, FGFR1, JAK2, CDKN2A, GNAQ, ABL1, NOTCH1.</p>
</blockquote>
<h3 id="2-relevant-variant-details">2. Relevant Variant Details</h3>
<table>
<thead>
<tr>
<th>Gene</th>
<th>Amino Acid Change</th>
<th>Coding</th>
<th>Variant Type</th>
<th>Allele Frequency</th>
<th>Transcrpit</th>
<th>Variant effect</th>
<th>ClinVar Significance</th>
</tr>
</thead>
<tbody>
<tr>
<td>ATM_16</td>
<td>p.T3024I</td>
<td>c.9071C&gt;T</td>
<td>SNP</td>
<td>0.5</td>
<td>ENST00000452508.6_1</td>
<td>MISSENSE</td>
<td>Uncertain_significance</td>
</tr>
<tr>
<td>KRAS_1</td>
<td>p.G13D</td>
<td>c.38G&gt;A</td>
<td>SNP</td>
<td>0.5</td>
<td>ENST00000256078.10_4</td>
<td>MISSENSE</td>
<td>Pathogenic/Likely_pathogenic</td>
</tr>
<tr>
<td>TP53_2</td>
<td>p.P72R</td>
<td>c.215C&gt;G</td>
<td>SNP</td>
<td>1.0</td>
<td>ENST00000269305.8_4</td>
<td>MISSENSE</td>
<td>drug_response</td>
</tr>
<tr>
<td>PIK3CA_9SELF</td>
<td>p.G914R</td>
<td>c.2740G&gt;A</td>
<td>SNP</td>
<td>0.5</td>
<td>ENST00000263967.4_2</td>
<td>MISSENSE</td>
<td>Pathogenic/Likely_pathogenic</td>
</tr>
</tbody>
</table>
<h2 id="citations">Citations</h2>
<blockquote>
<p>Disclaimer: The data presented here is from a curated knowledgebase of publicly available information, but may not be exhaustive.</p>
</blockquote>

        
        
    </body>
    </html>