# Pacific Informatics</br>Gene Panel DNAseq variant analysis

**_PHYSICIAN REPORT_**

---

## Table of Contents

- [Pacific InformaticsGene Panel DNAseq variant analysis](#pacific-informaticsgene-panel-dnaseq-variant-analysis)
  - [Table of Contents](#table-of-contents)
  - [Sample report](#sample-report)
    - [1. Genes Assayed](#1-genes-assayed)
    - [2. Relevant Variant Details](#2-relevant-variant-details)
  - [Citations](#citations)

## Sample report

***Date: April 28th 2023***

### 1. Genes Assayed

**_Genes Assayed for the Detection of DNA Sequence Variants_**

>MPL, NRAS, RET, PTEN, FGFR2, HRAS, ATM, KRAS, PTPN11, HNF1A, FLT3 ,BRCA2, RB1, AKT1, IDH2, TSC2, CDH1, TP53, ERBB2, BRCA1, SMAD4, STK11, GNA11, JAK3, ALK, IDH1, ERBB4, SRC, GNAS, SMARCB1, VHL, MLH1, CTNNB1, PIK3CA, FGFR3, PDGFRA, KIT, KDR, FBXW7, APC, CSF1R, NPM1, EGFR, MET, SMO, BRAF, EZH2, FGFR1, JAK2, CDKN2A, GNAQ, ABL1, NOTCH1.

### 2. Relevant Variant Details

| Gene | Amino Acid Change | Coding | Variant Type | Allele Frequency | Transcrpit | Variant effect | ClinVar Significance |
| --- | --- | --- | --- | --- | --- | --- | --- |
| ATM_16 | p.T3024I | c.9071C>T | SNP | 0.5 | ENST00000452508.6_1 | MISSENSE | Uncertain_significance |
| KRAS_1 | p.G13D | c.38G>A | SNP | 0.5 | ENST00000256078.10_4 | MISSENSE | Pathogenic/Likely_pathogenic |
| TP53_2 | p.P72R | c.215C>G | SNP | 1.0 | ENST00000269305.8_4 | MISSENSE | drug_response |
| PIK3CA_9SELF | p.G914R | c.2740G>A | SNP | 0.5 | ENST00000263967.4_2 | MISSENSE | Pathogenic/Likely_pathogenic |

## Citations

> Disclaimer: The data presented here is from a curated knowledgebase of publicly available information, but may not be exhaustive.
